multisig/tests/multisig_blackbox_test.rs

```diff
@@ -896,17 +1051,94 @@ fn ethereum_to_multiversx_tx_batch_rejected_test() {
         .run();
 }
 
+#[test]
+fn init_test() {
+    let mut state = MultiTransferTestState::new();
+
+    let mut board: MultiValueEncoded<StaticApi, ManagedAddress<StaticApi>> =
+        MultiValueEncoded::new();
+    board.push(ManagedAddress::from(RELAYER1_ADDRESS.eval_to_array()));
```

Comment:
I think it's easier to use the `to_managed_address` method
`RELAYER1_ADDRESS.to_managed_address()`

Reply:
thanks for the suggestion, indeed easier